build.sbt

```diff
@@ -804,7 +805,13 @@ lazy val `scio-parquet`: Project = project
     description := "Scio add-on for Parquet",
     libraryDependencies ++= Seq(
       "org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionCompatVersion,
-      "me.lyh" %% "parquet-avro" % parquetExtraVersion,
+      "me.lyh" %% "parquet-avro" % parquetExtraVersion excludeAll (
+        // parquet-avro depends on avro 1.10.x
```

Comment:
@nevillelyh hopefully this will work ok as it doesn't use any of the impact API. 🤞

Reply:
Right I need to make those Avro deps provided. https://github.com/nevillelyh/parquet-extra/issues/123